src/DispatchFunction.ts

```diff
@@ -1,7 +1,7 @@
 import ActionFunction from './ActionFunction';
 
 interface DispatchFunction {
-    (action: ActionFunction, actionType: string, args: IArguments): Promise<any> | void;
+    (action: ActionFunction, actionType: string, args: IArguments,  actionContext: { [key: string]: any }): Promise<any> | void;
```

Comment:
I didn't think about how many places this type would show up...can you add a file for ActionContext and define an interface for it there?  (Similar to how ActionFunction is used.)